### PR TITLE
Add content share integration test

### PR DESCRIPTION
**Pull request by ltrung**
*Description of changes*
Add a e2e test for content sharing.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: integration/configs/content_share_screen_capture_test.config.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  ],
+  "tests": [
+    {
+      "name": "Content Share Screen Capture %ts",
+      "description": "Screen sharing using content share",
+      "tupleSize": 1,
+      "noOfThreads": 1,
+      "testImpl": "ContentShareScreenCapture.js",
+      "payload": {
+        "url": "https://xijghn8zxf.execute-api.us-east-1.amazonaws.com/Prod/v2",
```
Comment (RidipDe) [inline integration/configs/content_share_screen_capture_test.config.json]: Shouldn't the url be set to "http://localhost:8080/" like in other config.json files?

File: integration/js/pages/AppPage.js [hunk 2]
```diff
@@ -95,6 +97,16 @@ class AppPage {
     await screenViewButton.click();
   }
 
+  async clickContentShareButton() {
+    let contentShareButton = await this.driver.findElement(elements.contentShareButton);
```
Comment (RidipDe) [inline integration/js/pages/AppPage.js]: contentShareButton can be const as it is not being changed

File: integration/js/pages/AppPage.js [hunk 3]
```diff
@@ -95,6 +97,16 @@ class AppPage {
...
+    await contentShareButton.click();
+  }
+
+  async clickContentSharePauseButton() {
+    let contentSharePauseButton = await this.driver.findElement(elements.contentSharePauseButton);
```
Comment (RidipDe) [inline integration/js/pages/AppPage.js]: contentSharePauseButton can be const as it is not being changed

File: integration/js/checks/ContentShareVideoCheck.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
+const {KiteTestError, Status} = require('kite-common');
+const AppTestStep = require('../utils/AppTestStep');
+
+class ContentShareVideoCheck extends AppTestStep {
+  constructor(kiteBaseTest, sessionInfo, testType, videoIndex) {
+    super(kiteBaseTest, sessionInfo);
+    this.expectedState = testType === 'ON' ? 'video' : 'blank';
```
Comment (RidipDe) [inline integration/js/checks/ContentShareVideoCheck.js]: If switch statement below is doing the same job, then why do we need this statement?

File: integration/js/checks/ContentShareVideoCheck.js [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    await step.execute(KiteBaseTest);
+  }
+
+  stepDescription() {
+    return 'Check the content share video in video index ' + this.videoIndex + " is " + this.expectedState;
+  }
+
+  async run() {
+    try {
+      let result = await this.page.videoCheck(this, this.videoIndex, this.expectedState);
```
Comment (RidipDe) [inline integration/js/checks/ContentShareVideoCheck.js]: result can be constant, its value is not changing

File: integration/configs/app_quit_audio_test.config.json [hunk 6]
```diff
@@ -27,7 +27,7 @@
     },
     {
       "browserName": "chrome",
-      "version": "77",
```
Comment (XHatan) [inline integration/configs/app_quit_audio_test.config.json]: Are there any issues with chrome 77 ?

**Comment by aburnson** [comment]
Typo in commit message and PR "integrationt"

**Comment by aburnson** [comment]
Rebased

**Pull request merged**